### ISSUE TITLE
Input does not accept defined Stylesheet (only direct style definition)

### ISSUE THREAD

**Issue by scstan** [GeekyAnts/NativeBase#115]
This does not change the input's text color:
```
<Input style={styles.textInput} placeholder="Email" />
...
const styles = StyleSheet.create({
  textInput: {
      color: styling.themeColor
    }
})
```
This changes the input color:
```
<Input style={{color: styling.themeColor}} placeholder="Email" />
```

**Comment by sankhadeeproy007** [comment]
Thanks for pointing out. Will look into it.

**Comment by sankhadeeproy007** [comment]
Fixed with 0.5.5
